addons/linecompass/functions/fnc_getUnitIcon.sqf

```diff
@@ -0,0 +1,28 @@
+#include "..\script_component.hpp"
+
+params ["_unit", "_player"];
+
+if !(_unit isNil QGVAR(customIcon)) exitWith {
+    _unit getVariable QGVAR(customIcon);
```

Comment:
i feel that needs to be combined with the radar custom icon somehow.

Reply:
then we would break backwards compat as i need a size for it

Reply:
and also the icons are a different style

Reply:
alight. Gotta add that to the wiki after the merge then.